.github/workflows/build.yml

```diff
@@ -60,11 +60,22 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Set up Go 1.19
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19.9
+          cache: false
+
       - name: Unit test
         run: make test
 
       - name: Upload Coverage Report
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v3  
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This token does not exist for this repo. Is this required?

Reply:
yes, should add a codecov token in this project.